co2calculator/distances.py

```diff
@@ -493,45 +494,40 @@ def create_distance_request(
     """
 
     # Validate the spatial data wrt. the mode of transportation
-    # NOTE: Since `start` and `destination` are different depending on the `transportation_mode`,
-    # we map the respective models to the mode of transportation.
-    # NOTE: Eventually the user of co2calculator should use the models right away (improve their
-    # structure in that iteration too!).
+    # And creates a DistanceRequest object containing either StructuredLocation,
+    # TrainStation or Airport objects based on the unser input address_type
+    # for start and destination, if nothing is given assume StructuredLocation
 
     try:
-        if transportation_mode in [
-            TransportationMode.CAR,
-            TransportationMode.MOTORBIKE,
-            TransportationMode.BUS,
-            TransportationMode.FERRY,
-            TransportationMode.BICYCLE,
-            TransportationMode.PEDELEC,
-        ]:
-            return DistanceRequest(
-                transportation_mode=transportation_mode,
-                start=StructuredLocation(**start),
-                destination=StructuredLocation(**destination),
-            )
-
-        if transportation_mode in [TransportationMode.TRAIN, TransportationMode.TRAM]:
-            return DistanceRequest(
-                transportation_mode=transportation_mode,
-                start=TrainStation(**start),
-                destination=TrainStation(**destination),
-            )
+        location = [None, None]
```

Comment:
```suggestion
        locations = [None, None]
```

Might be more logical to call it `locations` because it contains start and destination location.